### PR TITLE
GPII-830: Onboard ZoomText 2019

### PR DESCRIPTION
**Pull request by JavierJF**
**Warning: Not merge**

This pull request is WIP and is actually in DEMO state.

### CODE REVIEW

File: provisioning/data/ZoomText/zoomtext_api.js [hunk 1]
```diff
@@ -0,0 +1,435 @@
+var fs = new ActiveXObject("Scripting.FileSystemObject");
```
Comment (javihernandez) [inline provisioning/data/ZoomText/zoomtext_api.js]: We are no longer using this "thing", right? If so, this entire file can be removed.

File: gpii/node_modules/processHandling/processHandling.js [hunk 2]
```diff
@@ -272,8 +272,13 @@ gpii.windows.closeProcessByName = function (filename, options) {
         var windowPid = ptr.deref();
 
         if (pids.indexOf(windowPid) !== -1) {
-            // Send WM_QUIT, which tells the thread to terminate.
-            gpii.windows.user32.PostMessageW(hwnd, windows.API_constants.WM_QUIT, options.exitCode, 0);
+            if (options.message === "WM_CLOSE") {
+                // Send WM_CLOSE, which tells the application to exit nicely.
+                gpii.windows.user32.PostMessageW(hwnd, windows.API_constants.WM_CLOSE, options.exitCode, 0);
```
Comment (javihernandez) [inline gpii/node_modules/processHandling/processHandling.js]: Maybe a question for @stegru too.
According to https://docs.microsoft.com/en-us/windows/desktop/winmsg/wm-close, 

> An application can prompt the user for confirmation, prior to destroying a window, by processing the WM_CLOSE message and calling the DestroyWindow function only if the user confirms the choice.

Can a user avoid us from stopping the app? Are we okay with this?

File: provisioning/data/ZoomText/zoomtext_api.js (hunk 1 above)
Comment (JavierJF) [reply provisioning/data/ZoomText/zoomtext_api.js]: Done

File: gpii/node_modules/processHandling/processHandling.js (hunk 2 above)
Comment (JavierJF) [reply gpii/node_modules/processHandling/processHandling.js]: @javihernandez yes, this is an approach we have taken with another ATs too, it's already in master and it's used for R&W, and will be used for JAWS. As far as we know, this is the recommended way to go by the ATs manufacturers.

**Comment by JavierJF** [comment]
```
}
08:54:47.977:  jq: FAIL: Module "gpii.tests.acceptance.windows.builtIn.config tests" Test name "Testing os_common - magnifier running on startup" - Message: Confirming initial state - at sequence position 9 of 30
08:54:47.977:  jq: Expected: {
    "gpii.windows.enableRegisteredAT": {
        "com.microsoft.windows.magnifier": [
            {
                "settings": {
                    "running": true
                }
            }
        ],
        "com.microsoft.windows.onscreenKeyboard": [
            {
                "settings": {
                    "running": false
                }
            }
        ]
    }
}
08:54:47.977:  jq: Actual: {
    "gpii.windows.enableRegisteredAT": {
        "com.microsoft.windows.magnifier": [
            {
                "settings": {
                    "running": undefined
                }
            }
        ],
        "com.microsoft.windows.onscreenKeyboard": [
            {
                "settings": {
                    "running": false
                }
            }
        ]
    }
}
```
Unrelated failure caused by ATs not closing.

**Pull request closed without merging**